pkg/exchange/okex/okexapi/get_open_orders_request.go

```diff
@@ -0,0 +1,112 @@
+package okexapi
+
+import (
+	"time"
+
+	"github.com/c9s/requestgen"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+//go:generate -command GetRequest requestgen -method GET -responseType .APIResponse -responseDataField Data
+//go:generate -command PostRequest requestgen -method POST -responseType .APIResponse -responseDataField Data
+
+type OpenOrder struct {
+	AccumulatedFillSize fixedpoint.Value `json:"accFillSz"`
+	// If none is filled, it will return "".
+	AvgPrice      fixedpoint.Value           `json:"avgPx"`
+	CreatedTime   types.MillisecondTimestamp `json:"cTime"`
+	Category      string                     `json:"category"`
+	ClientOrderId string                     `json:"clOrdId"`
+	Fee           fixedpoint.Value           `json:"fee"`
+	FeeCurrency   string                     `json:"feeCcy"`
+	// Last filled time
+	FillTime       types.MillisecondTimestamp `json:"fillTime"`
+	InstrumentID   string                     `json:"instId"`
+	InstrumentType InstrumentType             `json:"instType"`
+	OrderId        types.StrInt64             `json:"ordId"`
+	OrderType      OrderType                  `json:"ordType"`
+	Price          fixedpoint.Value           `json:"px"`
+	Side           SideType                   `json:"side"`
+	State          OrderState                 `json:"state"`
+	Size           fixedpoint.Value           `json:"sz"`
+	TargetCurrency string                     `json:"tgtCcy"`
+	UpdatedTime    types.MillisecondTimestamp `json:"uTime"`
+
+	// Margin currency
+	// Only applicable to cross MARGIN orders in Single-currency margin.
+	Currency string `json:"ccy"`
+	TradeId  string `json:"tradeId"`
+	// Last filled price
+	FillPrice fixedpoint.Value `json:"fillPx"`
+	// Last filled quantity
+	FillSize fixedpoint.Value `json:"fillSz"`
+	// Leverage, from 0.01 to 125.
+	// Only applicable to MARGIN/FUTURES/SWAP
+	Lever string `json:"lever"`
+	// Profit and loss, Applicable to orders which have a trade and aim to close position. It always is 0 in other conditions
+	Pnl          string `json:"pnl"`
+	PositionSide string `json:"posSide"`
+	// Options price in USDOnly applicable to options; return "" for other instrument types
+	PriceUsd string `json:"pxUsd"`
+	// Implied volatility of the options orderOnly applicable to options; return "" for other instrument types
+	PriceVol string `json:"pxVol"`
```

Comment:
decided not to use fixedpoint.Value?

Reply:
Since the field is only used in options, I ignore it. XD

i can update it